packages/data-point/lib/entity-types/entity-collection/resolve.test.js

```diff
@@ -94,14 +94,14 @@ describe('entity.collection.filter', () => {
     })
   })
 
-  test('should resolve filter Transform', () => {
+  test('should resolve filter Transform | values of collection 3 passed into acc', () => {
```

Comment:
Can we add `It` to the beginning of this description? I don't think we need the stuff from the `|` on.